src/js/components/Carousel/Carousel.js

```diff
@@ -112,7 +112,10 @@ const Carousel = ({
         // eslint-disable-next-line react/no-array-index-key
         key={index}
         icon={
-          <CurrentIcon color={activeIndex === index ? iconColor : undefined} />
+          <CurrentIcon
+            color={activeIndex === index ? iconColor : undefined}
+            a11yTitle={`Slide ${index + 1}`}
```

Comment:
I think we should be putting the `a11yTitle` on the icon instead of on the Button

Reply:
If we put the `a11yTitle` on the button then the icon will have `Subtract Button` a11yTitle by default.

Reply:
Agree with @ericsoderberghp that the `a11yTitle` should be at the Button level. 

In my testing, if the `a11yTitle` is placed on the Button, the text is read correctly as "Slide 1, button" and the "subtract, button" is no longer read.

Reply:
@halocline Okay, I will change it. Thank you for your help guys.